### PR TITLE
Doc updates to cookbooks: rightscale, sys, sys_dns, sys_firewall, sys_ntp, web_apache

### PR DESCRIPTION
**Pull request by deanonishi**
Minor doc updates

### CODE REVIEW

File: cookbooks/repo/metadata.rb [hunk 1]
```diff
@@ -14,40 +14,40 @@
...
+  :display_name => "Repository URL",
+  :description => "The URL that points to the location of the repository that contains the application code. Specify a read-only URL. Example: http://mysvn.net/app/ or git://github.com/username/myapp.git",
   :required => "recommended",
   :recipes => ["repo::default"]
 
 attribute "repo/default/revision",
-  :display_name => "Branch/Tag",
-  :description => "Enter the branch of your repository you want to fetch. Example: master",
+  :display_name => "Branch/Tag/SHA",
+  :description => "The specific branch, tag, or SHA of the specified git repository that the application code will be retrieved from. Use 'master' to retrieve the master branch from the repository. Example: mybranch",
```
Comment (douglaswth) [inline cookbooks/repo/metadata.rb]: This is generic so it should not say SHA (in fact neither should we!); it should say commit instead.

File: cookbooks/repo_git/README.rdoc [hunk 2]
```diff
@@ -3,11 +3,11 @@
 == DESCRIPTION:
 
 Provides the GIT implementation of the 'repo' resource to
-manage source code download from GIT repositories.
+manage source code downloaded from git repositories.
```
Comment (douglaswth) [inline cookbooks/repo_git/README.rdoc:6]: Please capitalize the first letter of Git when it is not being used to specify the command.

File: cookbooks/repo_ros/README.rdoc [hunk 3]
```diff
@@ -18,16 +18,16 @@ Please see <tt>metadata.rb</tt> file for the latest dependencies.
 
 == KNOWN LIMITATIONS:
 
-Supports only Amazon S3 bucket and Rackspace cloud files.
+Cookbook only supports the retrieval of code from Amazon S3 and Rackspace Cloud Files.
```
Comment (douglaswth) [inline cookbooks/repo_ros/README.rdoc]: Which cookbook? Maybe this cookbook?

File: cookbooks/repo_ros/README.rdoc [hunk 4]
```diff
@@ -36,17 +36,16 @@ To access to Remote Object Store repositories all inputs shown below must be fil
...
 
 === General
 
-The 'repo_ros' implementation can be called with help of Lightweight Resource, which can be
+The 'repo_ros' implementation can be called with help from the Lightweight Resource, which can be
 found in the <tt>repo</tt> cookbook.
 
-For more about Lightweight Resources and Providers (LWRPs), please see the RightScale
-Guide at:  http://support.rightscale.com/12-Guides/Chef_Cookbooks_Developer_Guide/08-Chef_Development/Lightweight_Resources_and_Providers_(LWRP)
+For more information about Lightweight Resources and Providers (LWRPs), please see {Lightweight Resources and Providers}[http://support.rightscale.com/12-Guides/Chef_Cookbooks_Developer_Guide/08-Chef_Development/Lightweight_Resources_and_Providers_(LWRP)]
```
Comment (douglaswth) [inline cookbooks/repo_ros/README.rdoc]: This can have a period at the end.

Comment (deanonishi) [inline cookbooks/repo_ros/README.rdoc]: yes, agreed

File: cookbooks/repo_ros/README.rdoc (hunk 3 above)
Comment (deanonishi) [inline cookbooks/repo_ros/README.rdoc]: I think both 'the' or 'this' would work.

**Comment by douglaswth** [comment]
This was actually merged.

**Comment by deanonishi** [comment]
That's actually awesome!

On Thu, Jun 21, 2012 at 4:41 PM, Douglas Thrift <
reply@reply.github.com

> wrote:
> 
> This was actually merged.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> 
> https://github.com/rightscale/rightscale_cookbooks/pull/125#issuecomment-6497416

## 

---

Dean Onishi
Sr. Technical Writer
RightScale Inc.
www.rightscale.com
dean@rightscale.com
805.500.4164 x1012 (work)
805.403.1045 (cell)
dean.onishi (skype)

Follow us on Twitter: www.twitter.com/RightScale
Connect with us on LinkedIn: http://linkd.in/rightscale
Join our fans on Facebook: www.facebook.com/RightScale

**Pull request closed without merging**